services/src/api/model/datatypes.rs

```diff
@@ -1485,3 +1485,29 @@ impl From<MultiPolygon> for geoengine_datatypes::primitives::MultiPolygon {
         .unwrap()
     }
 }
+
+#[derive(PartialEq, Eq, Serialize, Deserialize, Debug, Clone)]
+pub struct StringPair((String, String));
```

Comment:
why did you change the concrete type like `GdalConfigOption`, to this generic type? Maybe it would make sense to have a type alias `type GdalConfigOption = StringPair`? I think it would lead to a better api doc.

Reply:
I wanted to use this type in multiple, similar places without having to duplicate the necessary code. I also used it in SpatialReferenceSpecification for he axis labels. But creating an alias is a great idea

Reply:
[8a1fa0e](https://github.com/geo-engine/geoengine/pull/666/commits/8a1fa0e0e4b06ffc4e22032561d3a0ad3d607bd4)